docs/about/release-notes.rst

```diff
@@ -27,6 +27,8 @@ Deprecations
 Stability, Maintainability, and Testing
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+* Added new tutorial: *RHESSI Solar Flares* `2536 <https://github.com/scipp/scipp/pull/2536>`_.
```

Comment:
Do we need a new Documentation section, or should we add Documentation to that section title?

Reply:
I think a new section makes sense. Especially when there are 'breaking' changes.

Reply:
@SimonHeybrock ?

Reply:
New sections sounds reasonable!